docs/tls/index.html

```diff
@@ -884,6 +884,8 @@ <h3 id="example">Example</h3>
 apiVersion: networking.x-k8s.io/v1alpha1
 metadata:
   name: my-app
+  annotations:
+    networking.x-k8s.io/app-protocol: https
```

Comment:
Nit: This doesn't match your example in the godocs above.

Reply:
Good catch. Updated.